pkgs/development/python-modules/gpy/default.nix

```diff
@@ -47,6 +51,11 @@ buildPythonPackage rec {
     done
   '';
 
+  disabledTests = lib.optionals (stdenv.isDarwin && stdenv.hostPlatform.isx86_64) [
+    # Breaks on Darwin x86_64 only
```

Comment:
Could you change this comment to explain instead what sort of failure results, as done in many Nix packages? Simply saying it's an issue on x64-darwin is fairly obvious given the code, and less helpful for e.g. evaluating the state of a package on a platform.

Reply:
Thsnk you. Let meknow if this works for you.